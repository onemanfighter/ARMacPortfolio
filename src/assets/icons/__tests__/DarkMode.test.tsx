import { render } from '@testing-library/react';
import { DarkModeIcon } from '../DarkMode';

describe('DarkMode', () => {
  it('should render correctly', () => {
    const { container } = render(<DarkModeIcon />);

    expect(container).toMatchSnapshot();
  });
});
