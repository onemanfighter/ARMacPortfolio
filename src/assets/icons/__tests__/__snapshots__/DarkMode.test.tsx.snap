// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DarkMode should render correctly 1`] = `
<div>
  <svg
    fill="none"
    viewBox="0 0 24 24"
  >
    <path
      d="M12 16a4 4 0 000-8v8z"
      fill="currentColor"
    />
    <path
      clip-rule="evenodd"
      d="M12 2C6.477 2 2 6.477 2 12s4.477 10 10 10 10-4.477 10-10S17.523 2 12 2zm0 2v4a4 4 0 100 8v4a8 8 0 100-16z"
      fill="currentColor"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;
